lib/mrsk/cli/build.rb

```diff
@@ -17,9 +17,15 @@ def push
       verify_local_dependencies
       run_hook "pre-build"
 
+      if (uncommitted_changes = Mrsk::Utils.uncommitted_changes).present?
+        say "The following paths have uncommitted changes (check your .gitignore file):\n #{uncommitted_changes}", :yellow
```

Comment:
You might also want to add the files to your `.dockerignore` to avoid them ending up in the image. I'm not sure we can get this down to a succinct message, so I think it's best to remove the `(check your .gitignore file)` from the message.

Reply:
I also been thinking about `.dockerignore`, but message was too big with it.